main/zoe/guide/contracts/simple-exchange.md

```diff
@@ -2,36 +2,36 @@
 
 <Zoe-Version/>
 
-##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/f29591519809dbadf19db0a26f38704d87429b89/packages/zoe/src/contracts/simpleExchange.js) (Last updated: 9/12/2020)
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/f29591519809dbadf19db0a26f38704d87429b89/packages/zoe/src/contracts/simpleExchange.js) (Last updated: 12-SEP-2020)
```

Comment:
for dates, I actually prefer 2020-09-12. (You're right that I used the US-only format. shame on me.)  It seems to be an ISO standard, and it seems to be uniquely recognizable both US and internationally.

Reply:
Makes sense. Used 12-SEP-2020 because that's what's currently on the site wide variable (changed from our previous 9/12/2020 style, so it wasn't just you). I can argue for why day-month is usually the most relevant info first in these cases, but ISO standard beats that.